recipes/megaman/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.1.1" %}
+
+package:
+    name: megaman
+    version: {{ version }}
+
+source:
+    fn: megaman-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/m/megaman/megaman-{{ version }}.tar.gz
+    md5: 924d05877a3bcfc86d4f48ea9436cdd4
+
+build:
+    number: 0
+    skip: true    # [win]
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - cython
+        - flann
+        - gcc 4.8*  # [osx]
```

Comment:
Yeah, I think he just wants to maintain ABI compatibility, which is ok. This is really the only version Continuum provides. Personally, I'm ok with this.